src/vma/dev/net_device_val.cpp

```diff
@@ -222,9 +222,7 @@ net_device_val::net_device_val(void *desc) : m_lock("net_device_val lock")
 	m_local_addr    = ((struct sockaddr_in *)ifa->ifa_addr)->sin_addr.s_addr;
 	m_netmask       = ((struct sockaddr_in *)ifa->ifa_netmask)->sin_addr.s_addr;
 
-	char base_ifname[IFNAMSIZ];
-	get_base_interface_name((const char*)(ifa->ifa_name), base_ifname, sizeof(base_ifname));
-	if (check_device_exist(base_ifname, BOND_DEVICE_FILE)) {
+	if (check_device_exist(m_base_name, BOND_DEVICE_FILE)) {
 		// this is a bond interface (or a vlan/alias over bond), find the slaves
```

Comment:
There are 3 more locations which we can replace with the new call : 
 
> [lirano@r-aa-apollo03 10:32:56 libvma]$ grep -nir "char base_ifname" src/vma/dev/net_device_val.cpp -A 1
380:            char base_ifname[IFNAMSIZ];
381-            if (get_base_interface_name((const char*)m_slaves[i]->if_name, base_ifname, sizeof(base_ifname))) {
--
1118:   char base_ifname[IFNAMSIZ];
1119-   get_base_interface_name((const char*)(ifa->ifa_name), base_ifname, sizeof(base_ifname));
--
1242:   char base_ifname[IFNAMSIZ];
1243-   get_base_interface_name((const char*)(ifname), base_ifname, sizeof(base_ifname));